packages/lib/versionInfo.test.ts

```diff
@@ -0,0 +1,62 @@
+import { _ } from './locale';
+import { getPluginsArr, PluginInfo } from './versionInfo';
+import PluginService from '@joplin/lib/services/plugins/PluginService';
+import Plugin from '@joplin/lib/services/plugins/Plugin';
+
+describe('versionInfo', function() {
+
+	it('should not push plugin to body if no plugin is found',() => {
```

Comment:
I don't understand what this test is doing.
If it's checking `getPluginsArr` then why do you need the rest remaining code ('if' and 'map')? If it's checking the logic inside the `if` then what's point of checking it when this code is only used here in this test?

Reply:
Fair point, I'll remove that too.